djhtml/modes.py

```diff
@@ -323,6 +331,20 @@ def create_token(self, raw_token, src):
         kind = "javascript"
         self.next_mode = self
 
+        if raw_token.lstrip().startswith("switch"):
```

Comment:
This will also be True for `function switcharoo()`

Reply:
Solved in [commit](https://github.com/rtts/djhtml/pull/49/commits/0086311c0b6e8aa360d343f34fe5ecf6f66c4d4d)